### DISCUSSION TITLE
[Feature request] - Manually mark snap as seen?

### DISCUSSION THREAD

**Discussion by Pandasapling** [rodit/SnapMod#146] [Ideas & Feature Suggestions]
It’s not entirely important but it’s probably something I’d use. 

Great work on the mod nonetheless!

**Comment by rodit** [comment]
I'm not quite sure what you mean? Surely it's easier to just open the snap.

**Reply by Pandasapling** [reply]
Basically I mean having the “Hide snap views/stealth mode” setting with a button or something over the snap so I could mark as seen from the snap view. If that makes sense?

**Reply by RevealedSoulEven** [reply]
If you'll go on holding every every user and click mark as seen, then better open the snaps🤣🤣 it's consumes the same man power. Useless feature bro. But yeaa if you want to view all snaps from all users at once, then we can consider it an issue.

**Reply by rodit** [reply]
It's actually quite difficult to mark chats/snaps as seen only locally (especially considering when you refresh the feed, they will be synced with the server and marked as unseen again). So, unfortunately, this probably isn't a feature I will implement.

**Reply by Pandasapling** [reply]
> It's actually quite difficult to mark chats/snaps as seen only locally (especially considering when you refresh the feed, they will be synced with the server and marked as unseen again). So, unfortunately, this probably isn't a feature I will implement.

 I was more meaning leaving it on unopened on both parties, being able to open it as many times without changing the fact it’s “unopened” then being able to mark as seen when you feel like telling the person you’ve opened it. 
 
It’s something I’ve seen in other mods and was something I personally used, however if it’s not something you can implement that’s completely okay:)

Thanks for your time and effort you’ve put into this mod, many people appreciate it!
